doc/DOCUMENTATION.org

```diff
@@ -550,6 +551,21 @@ change the location of this directory.
 so =~/.spacemacs= must not exist for =~/.spacemacs.d/init.el= to be used by
 Spacemacs.
 
+*** Chemacs users with =$CONFIG_HOME/.spacemacs.d= as =$SPACEMACSDIR=
+Users of Chemacs may use =$CONFIG_HOME/.spacemacs.d= as =$SPACEMACSDIR=, so long
+as they define an additional environment variable =SPACEMACS_INIT_FILE_NAME= to
+something other than ~init.el~. Chemacs users can still use =.spacemacs.d= as
+their dotfile directory with Chemacs as long as they retain a =.spacemacs= file
+otherwise, if they do not declare an alternative dotfile name and locate that in
+the dotfile directory.
+
+#+name: chemacs-spacemacsdir-example
```

Comment:
what does this code snippet mean?
where should it be added?
it's not correctly indented.

Reply:
It's only an example for Chemacs users. If a user is not using Chemacs, is using `.spacemacs.d`, and doesn't want to use `~/.spacemacs` then they will need to declare the environment variable in their shell.

I can improve the clarity of the documentation and code comments.

Reply:
For Chemacs users, this is an example of the `.emacs-profiles.el` file.

Reply:
> it's not correctly indented.
Are you looking at 26c054394c727a7b868c1d60e19b2b51f3ec82e5? I already changed the indentation of the example last night.